cmd/root.go

```diff
@@ -20,8 +20,8 @@ var rootCmd = &cobra.Command{
 configuration checks using Open Policy Agent (OPA).
 
 Preflight checks are bundled into Packages`,
-	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		logs.Initialize()
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		return logs.Initialize()
 	},
```

Comment:
Returning the error here means that the error can be logged in the same format as all the other errors that are returned by the command.